pkg/vere/lord.c

```diff
@@ -10,49 +10,42 @@
 
 /*
 |%
-::  +writ: from king to serf
+::  peek=[gang (each path $%([%once @tas @tas path] [%beam @tas beam]))]
+::  ovum=ovum
 ::
-+$  writ
-  $%  $:  %live
-          $%  [%cram eve=@]
-              [%exit cod=@]
-              [%save eve=@]
-              [%meld ~]
-              [%pack ~]
-      ==  ==
-      [%boot cax=(list [k=[s=* f=*] v=*]) lit=(list ?((pair @da ovum) *))]
-      [%peek mil=@ sam=*]  :: gang (each path $%([%once @tas @tas path] [%beam @tas beam]))
-      [%play eve=@ lit=(list (pair @da ovum))]
+|$  [peek ovum]
+|%
+::  +task: from urth to mars
+::
+::    next steps:
+::    - |mass should be a query of the serf directly
+::    - add duct or vane stack for spinner
+::
++$  task
+  $%  [%live ?(%meld %pack) ~] :: XX rename
+      [%exit ~]
+      [%peek mil=@ peek]
+      [%poke mil=@ ovum]
       $:  %quiz
           $%  [%quac ~]
               [%quic ~]
       ==  ==
```

Comment:
You restored these IPC message types here in https://github.com/urbit/vere/commit/ba8e045cb7ef835994ba1fc7f12ef3bb35398961, and the related lord.c functions in https://github.com/urbit/vere/commit/d63ad694efea0b197d5855b67bc36926dde5cebe. But this protocol needs to be implemented on both sides -- those type haven't been copied over to mars.c, and the ipc messages in question are not handled.